### DISCUSSION TITLE
Squarespace

### DISCUSSION THREAD

**Discussion by joshferox1** [bullhorn/career-portal#490] [Q&A]
Has anybody gotten this to work on a Squarespace site? If so, how?!

**Comment by pbearne** [comment]
Hi @joshferox1 
I don't think you will be able to get it integrated into Squarespace. A solution would be to set it up on a subdomain (jobs.prime-domain.com) 
You have hit a limitation of a SAAS solution.

The real fix is to move off (to WordPress) to a solution you host where you have control/options.

Paul
matadorjobs.com
